src/mumble_exe/mumble_exe.cpp

```diff
@@ -125,8 +137,151 @@ static const std::wstring GetAbsoluteMumbleAppDllPath(std::wstring suggested_bas
 	return base_dir + L"\\mumble_app.dll";
 }
 
+// UIAccessDisabledViaConfig queries the configuration store, and returns
+// whether UIAccess has been disabled by the user via the "shortcuts/uiaccess"
+// config option.
+static bool UIAccessDisabledViaConfig() {
```

Comment:
I'd like to point out that this does not break portable builds, because portable builds will NOT be run from Program Files.

As such, they will never consider the return value of this function, because they return early via https://github.com/mkrautz/mumble/commit/11f2c558d3fadb4592757fb6a969b6ab995faf7f#diff-3edfe24f667fa5241aff2ba5697d6b8bR202, since the process will not have UIAccess if is not run from a "system" directory such as Program Files.